ingress/controllers/nginx/nginx/ssl.go

```diff
@@ -45,17 +45,28 @@ type SSLCert struct {
 
 // AddOrUpdateCertAndKey creates a .pem file wth the cert and the key with the specified name
 func (nginx *Manager) AddOrUpdateCertAndKey(name string, cert string, key string) (SSLCert, error) {
+	tempPemFileName := name + ".pem"
 	pemFileName := config.SSLDirectory + "/" + name + ".pem"
```

Comment:
maybe this could be `pemFileName := fmt.Sprintf("%v/%v", config.SSLDirectory, tempPemFileName)`